### DISCUSSION TITLE
in staxtrip 2.35 and 2.36 when encoding dv file it require vbv max and min value

### DISCUSSION THREAD

**Discussion by prtkm08** [staxrip/staxrip#1245] [Q&A]
![Screenshot 2024-02-24 114745](https://github.com/staxrip/staxrip/assets/37538032/8debdaab-c20d-40ce-9244-97c3cc8ce563)
look at screen shot and tell me me what value to entire
![Screenshot 2024-02-24 114909](https://github.com/staxrip/staxrip/assets/37538032/c73d2d31-13d0-4e0a-90f6-9d4e49be4b77)

**Comment by jessielw** [comment]
This is up to the user. It's based on what you need. 

There is defaults for it though. So look up the defaults for your level enter them if you don't feel like learning buffer.

**Reply by prtkm08** [reply]
can you tell me where is defaults i do'nt know where it is

**Comment by Dendraspis** [comment]
A table with the `Max Bitrate` can be found on Wikipedia:
https://en.wikipedia.org/wiki/High_Efficiency_Video_Coding_tiers_and_levels#Levels

Depending on your encoder settings the most common value would be `160000`, but you should verify that.
